src/modelldcatnotordf/modelldcatno.py

```diff
@@ -0,0 +1,303 @@
+"""Modelldcatno module for mapping a Modelldcatno model to rdf.
```

Comment:
Er tanken at alle dataklassene knyttet til modelldcatno skal inn i denne ene fila?

Reply:
Ja, det stemmer. Alle klassene i modellen må inn i en fil fordi det er klasser i modellen som har pekere til hverandre. Og det støttes foreløpig ikke av import-mekanismen i Python. Men andre klasser som ikke er en del av modellen kan selvfølgelig ligge utenfor

Reply:
Det kommer til å bli en veldig stor fil og det er bare et problem for ModelElement - ModelProperty

Reply:
Vi får se. Vi kan diskutere dette senere en gang kanskje når vi har litt bedre tid og mindre ting å gjøre. Men du får heller ta den praten med @Chlenix , fordi jeg tror han tross alt er mer dreven i Python og Python-standarder enn noen av oss andre. Du kan sjekke her hvordan andre folk gjør tilsvarende : https://github.com/psf/requests/blob/master/requests/models.py